app/components/Home/AllAmbassadors.jsx

```diff
@@ -1,37 +1,104 @@
 import React from 'react'
 import { Link } from 'react-router'
 import styled from 'styled-components'
-import { Flex } from '@rebass/grid'
-
-import imgBasile from '../../assets/ambassadors/basile.jpg'
-import imgDorian from '../../assets/ambassadors/dorian.jpg'
+import { Flex, Box } from '@rebass/grid'
+import { withNamespaces } from 'react-i18next'
+import { fontWeight } from 'styled-system'
+import imgBenjamin from '../../assets/ambassadors/benjamin.png'
+import imgFlorence from '../../assets/ambassadors/florence.jpg'
 import imgFrederic from '../../assets/ambassadors/frederic.jpg'
-import imgThimothee from '../../assets/ambassadors/timothee.jpg'
+import imgBasile from '../../assets/ambassadors/basile.jpg'
+import imgMathieu from '../../assets/ambassadors/mathieu.jpg'
 
 const AMBASSADORS = [
```

Comment:
We should rename the variables, filename and component from "Ambassador" to "Team" since we're now displaying everyone in there.